open-reac/src/main/java/com/powsybl/openreac/Reports.java

```diff
@@ -84,10 +86,33 @@ public static void createShuntModificationsReporter(ReportNode reportNode, Strin
                     .withMessageTemplate("shuntCompensatorDeltaDiscretizedOptimizedOverThreshold", "After discretization, shunt compensator ${shuntCompensatorId} with ${maxSectionCount} available section(s) has been set to ${discretizedValue} MVar (optimal value : ${optimalValue} MVar)")
                     .withUntypedValue("shuntCompensatorId", shunt.id())
                     .withUntypedValue("maxSectionCount", shunt.maximumSectionCount())
-                    .withUntypedValue("discretizedValue", REACTIVE_VALUE_FORMAT.format(shunt.discretizedReactiveValue()))
-                    .withUntypedValue("optimalValue", REACTIVE_VALUE_FORMAT.format(shunt.optimalReactiveValue()))
+                    .withUntypedValue("discretizedValue", VALUE_FORMAT.format(shunt.discretizedReactiveValue()))
+                    .withUntypedValue("optimalValue", VALUE_FORMAT.format(shunt.optimalReactiveValue()))
                     .withSeverity(TypedValue.TRACE_SEVERITY)
                     .add());
         }
     }
+
+    public static void reportVoltageLevelsWithLimitsOutOfNominalVRange(ReportNode reportNode, Map<String, VoltageLevelLimitInfo> voltageLevelsWithLimitsOutOfNominalVRange) {
+        if (!voltageLevelsWithLimitsOutOfNominalVRange.isEmpty()) {
+            ReportNode reportLimitsOutOfRange = reportNode.newReportNode()
+                .withMessageTemplate("voltageLevelsLimitsOutOfNominalVRange", "Voltage levels limits out of nominal voltage range")
+                .add();
+
+            reportLimitsOutOfRange.newReportNode()
```

Comment:
You can add a small comment to not change the key of the report

Reply:
:+1: 
Comment added